_configs/eval_ssm.yaml

```diff
@@ -68,6 +68,19 @@ infmax_models:
         max_nb_clusters: 10
         experiment_name: "My Experiment"
         visualise: False
+  - name: "GBIM"
+    class: "GBIM"
+    parameters:
+      rng_seed: 43
+      device: cpu
```

Comment:
[1/3] does it work on GPU? :D

Reply:
[2/3] I'd move `device` under `run` section and out here `auto` value (like e.g. `nb_seeds` in `MultiNode2VecKMeansAuto`), because there will be other models which utilise GPU, so this field can be shared

Reply:
It is passed to torch.device so it should work. I do not have available gpu lately to verify that